### ISSUE TITLE
Cannot set username

### ISSUE THREAD

**Issue by asiletto** [padams/Open-Web-Analytics#168]
I am trying to set the username on a tracked site, but on the console I don't see it

`owa_cmds.push(['setUserName', 'Mario']);
`

Watching the logs I see the username is sent only on event types **dom.click** and not on event types **base.page_request**

I am setting it correctly?

**Comment by chris4321** [comment]
I am having the same problem.  EVen though unique visitors are listed, I don't see any user id information on my OWA dashboard, even after adding owa_cmds.push(['setUserName', 'email@company.com']); to the tracker code.I haven't been able to find any documentation on what options are available for the ows_cmds.push calls, so I have no idea if this is correct, and it's based on one other mention I found online.  Is it even possible to get this type of information to show up on the OWA reports?

**Comment by coppai** [comment]
Ran into the same issue here.  We ended up rolling back to a previous version of OWA which resolved the problem.  Looks like the 'log_visitor_pii' config value always returns false, regardless of the default or db value.  

We also looked at modifying the owa_trackingEventHelpers.setUserName method.  There is a conditional there that checks the log_visitor_pii config value and won't set the username if it is false.  By removing that check, the username is always set correctly.  In the end though we didn't want to deviate from the original code.
